src/utils/urlUtils.js

```diff
@@ -0,0 +1,18 @@
+export function sanitizeTextFromDomains(text) {
+  if (!text) {
+    return '';
+  }
+  const trimmedText = text.slice(0, 30);
```

Comment:
why 30?

Reply:
No specific reason, in the audit report they just recommended to limit it.

Reply:
ah ok.. then i guess you took the 30 length based on the url that wallet connect provides? but do you think it would be safe, just because of this hard length if we have problem then have to make a release just to allow more length for valid URLs 😩